pkg/cvo/status.go

```diff
@@ -84,3 +128,64 @@ func checkForUpdate(config cvv1.CVOConfig) ([]cincinnati.Update, error) {
 	}
 	return cincinnati.NewClient(uuid).GetUpdates(string(config.Upstream), config.Channel, version.Version)
 }
+
+func SetOperatorStatusCondition(conditions *[]osv1.OperatorStatusCondition, newCondition osv1.OperatorStatusCondition) {
```

Comment:
Can you move these to `apis`

Reply:
> Can you move these to `apis`

No.  We don't put code in api packages.

Reply:
But these are tied to the api. they are definitely not part of the `pkg/cvo` package api.